migration-from-4x.md

```diff
@@ -1,66 +1,76 @@
+* No core editor methods were removed (such as: `tinymce`, `editor`, `selection`, and `on()`).
 
-### Methods
 
-* All TinyMCE 4.x methods for creating UI components have been removed. New methods have been added for TinyMCE 5.0. For more information, see the [docs]({{site.baseurl}}/ui-components/).
-* No core editor methods were removed (tinymce, editor, selection, on(), etc. remain the same).
+### Editor: Changes to Settings
 
-### Settings
+TinyMCE 5 includes the following changes to editor-core settings:
 
-In TinyMCE 5.0, some configurations have been removed because they are no longer necessary or an improved solution has been introduced.
+* [New Editor settings](#neweditorsettings)
+* [Changed Editor settings](#changededitorsettings)
+* [Removed Editor settings](#removededitorsettings)
 
-#### Added settings
+#### New Editor settings
 
 | **Setting** | **Description** |
 | ----------- | --------------- |
-| [`custom_colors`]({{site.baseurl}}/configure/content-appearance/#custom_colors) | **custom_colors** is `true` by default. When it is set to `false`, it will turn off the custom color picker in the color swatch. |
+| [`custom_colors`]({{site.baseurl}}/configure/content-appearance/#custom_colors) | **custom_colors** is `true` by default. Setting `custom_colors: false` removes the custom color picker in the color swatch. |
 
-#### Removed settings
+#### Changed Editor settings
 
-* [`file_browser_callback`]({{site.url}}/docs-4x/configure/file-image-upload/#file_browser_callback) - Previously, the `file_browser_callback` option was used to add a file or image browser to TinyMCE. This feature was deprecated in version 4.1.0 and replaced by [`file_picker_callback`]({{site.baseurl}}/configure/file-image-upload/#file_picker_callback). `file_browser_callback` option has been removed from TinyMCE 5.0 owing to the configuration changes in the dialog component.
+| **Setting**                                                      | **TinyMCE 4**                                                                                 | **TinyMCE 5**                                                                                                                                                              |
+| ---------------------------------------------------------------- | --------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [`height`]({{site.baseurl}}/configure/editor-appearance/#height) | Set the height of the editable area of the editor.<br/>Only accepted supported number values. | Sets the overall height of the editor, including the menubar, toolbars, and statusbar.<br/>Supports numbers and strings. Assumes strings are a valid CSS value for height. |
```

Comment:
is "accepted supported" the right phrase here?

Reply:
Revised to ```Supported number values.```